// quiz-backend/quizzes.js
const quizzes = [
  {
    id: "wellbeing-bachelors",
    title: "Well-Being in Practice Bachelors",
    description: "Well-being foundations and positive psychology",
    questions: [
      {
        text: "What are the 3 pillars of the IE Center for Health & Well-Being?",
        options: [
          "Body Balance, Party, and Religion",
          "Body Pump, Cross-fit, Zumba",
          "Body, Mind, Soul"
        ],
        correctIndex: 2
      },
      {
        text: "IE Center for Health & Well-Being is...",
        options: [
          "Balance, Appreciation, and Perspective",
          "Balance, Happiness, and Perspective",
          "Meaning, Thriving, and Positive State"
        ],
        correctIndex: 0
      },
      {
        text: "Positive Psychology is the scientific study of what makes life worth living...",
        options: [
          "False",
          "True"
        ],
        correctIndex: 1
      },
      {
        text: "Positive Psychology aims for individuals and communities...",
        options: [
          "Thriving",
          "Survival"
        ],
        correctIndex: 0
      },
      {
        text: "What is Resilience?",
        options: [
          "The capacity to recover quickly from difficulties",
          "The capacity to learn from difficulties",
          "The capacity to come out reinforced from difficulties",
          "All options are correct"
        ],
        correctIndex: 3
      },
      {
        text: "How do we build resilience?",
        options: [
          "By understanding our emotions",
          "By understanding our emotions' utility",
          "By understanding our emotions' impact on our well-being and behavior",
          "All options are correct"
        ],
        correctIndex: 3
      },
      {
        text: "Pleasant emotions...",
        options: [
          "Broaden our visual field",
          "Have a pro-social action response",
          "Create community",
          "All options are correct"
        ],
        correctIndex: 3
      },
      {
        text: "Unpleasant emotions...",
        options: [
          "Are connected to the upward spiral",
          "Essential for survival. They are signals",
          "Don't build resilience",
          "All options are correct"
        ],
        correctIndex: 1
      },
      {
        text: "I have agency to deal with my emotions.",
        options: [
          "True",
          "False"
        ],
        correctIndex: 0
      }
    ]
  },

  {
    id: "wellbeing-masters",
    title: "Well-Being in Practice Masters",
    description: "Advanced well-being and leadership",
    questions: []
  },

  {
    id: "bienestar-en-practica",
    title: "Bienestar en Práctica",
    description: "Bienestar y psicología positiva",
    questions: []
  },

  {
    id: "attention-management",
    title: "Attention Management for Learning",
    description: "Focus, attention, and learning habits",
    questions: []
  },

  {
    id: "strengths-based-mindset",
    title: "Strengths-Based Mindset",
    description: "Strengths development and growth mindset",
    questions: []
  },

  {
    id: "purposeful-decisions",
    title: "Purposeful Decisions for Life's Journey",
    description: "Decision-making and personal purpose",
    questions: []
  }
];

module.exports = { quizzes };
